doc/conf.py

```diff
@@ -137,6 +137,7 @@
     "logo_only": True,
     "prev_next_buttons_location": None
 }
+html_baseurl = "https://docs.zephyrproject.org/latest/"
```

Comment:
shouldn't this be changed to
```suggestion
html_baseurl = f"https://docs.zephyrproject.org/{version if is_release else 'latest'}/"
```

Reply:
Since released versions have the "disclaimer popup" saying "The latest development version of this page may be more current than this released x.y.z version." I would argue that latest should always be the canonical?

Reply:
right